packages/panels/src/components/page-header/PageHeader.tsx

```diff
@@ -17,9 +17,14 @@ export const PageHeader: React.FC<PageHeaderProps> = ({
   children,
 }) => {
   return (
-    <Box shadow={"box"} background={cssColor("--lhds-color-ui-50")}>
+    <Box shadow={"box"} background={cssColor("--lhds-color-ui-50")} gap={2}>
       <Box indent={3}>
-        {renderBreadCrumbs && <Row spacing={1.25}>{renderBreadCrumbs()}</Row>}
+        {renderBreadCrumbs && (
+          <Row height={"32px"} alignItems={"center"}>
+            <Row width={"64px"} />
```

Comment:
`<Space num={8} />`?

Reply:
Same here, the design is 64px, so it made more sense to specify it explicitly.